layouts/default.vue

```diff
@@ -13,25 +19,31 @@
         <mtl-navbar-top />
         <nuxt />
       </div>
-    </div>
-
-    <mtl-footer />
+    </div>-->
+    <!-- <mtl-footer /> -->
   </div>
 </template>
 
 <script>
+import { MtlNavMobile, MtlSideBarMobile } from '@/components/index'
 export default {
   components: {
-    MtlNavbarTop: () => import('~/components/navbar/navbar-top'),
-    MtlNavbarLeft: () => import('~/components/navbar/navbar-left'),
-    MtlFooter: () => import('~/components/footer')
+    MtlSideBarMobile,
+    MtlNavMobile
   },
+  data: () => ({
+    currentRoute: ''
+  }),
   computed: {
     openNav() {
       return this.$store.state.navMobile
     }
   },
+  watch: {
+    '$route.query': 'parseRoute'
```

Comment:
Vue trick:
```suggestion
    '$route.query': {
        handler: 'parseRoute',
        immediate: true,
      }
```
then you don't have to call `this.parseRoute()` in `mounted()`

Reply:
yep forget about it